lib/TelemetryLog.js

```diff
@@ -67,21 +58,51 @@ class TelemetryLog {
     });
   }
 
-  disableSite(domain) {
-    this.logData.disableCount += 1;
+  passwordChange(domain) {
+    this.logData.passwordChangeCount += 1;
+    this.log({
+      messageType: 'passwordChange',
+      domain,
+      passwordChangeConfirmTime: Date.now(),
+      passwordChangeCount: this.logData.passwordChangeCount,
+    });
+  }
+
+  noAccount(domain) {
```

Comment:
Instead of having so many types of events, couldn't you have something more generic? The addon could `this.port.emit('uiEvent', {type: 'noAccount'})`, and all of these listeners could probably be reduced to something like

```js
function logUiEvent(messageType) {
  this.logData[messageType].count += 1;
  this.log({
    messageType,
    domain: this.activeRecDomain,
    [messageType + 'ConfirmTime']: Date.now(),
    [messageType + 'Count']: this.logData[messageType].count,
  });
}    
```

This would mean fewer event handlers, and fewer logging functions. That means less code to read, and less code to change in the case of future refactors. If it is at least as readable, that makes it a net win to me. It also means adding a new type of event is much easier.